_episodes/14-modules.md

```diff
@@ -6,9 +6,10 @@ questions:
 - "How do we load and unload software packages?"
 objectives:
 - "Understand how to load and use a software package."
+- "Understand how the module mechanism interacts with the shell."
```

Comment:
```suggestion
- "Load and use a software package."
- "Explain how the module mechanism interacts with the shell."
```

I think this second one can be more specific TBH - interacts how? In what respect? In terms of affecting the PATH/environment? Or something else?

Reply:
Right, the exercise in the snippets shows changes to PATH after doing a `module load`. Perhaps:
> "Explain how the shell environment changes when the module mechanism loads or unloads packages."

Reply:
Wordy, but more meaningful 😁 

If you update the objective as you suggest, I would be happy to approve this PR

Reply:
Done! b3c4a2c